go/Dockerfile

```diff
@@ -1,17 +1,19 @@
-FROM ubuntu:14.04
+FROM alpine:3.3
 MAINTAINER Keybase <admin@keybase.io>
 
-RUN apt-get update && apt-get install -y fuse openssl jq
```

Comment:
Good point: `openssl` is no longer used on the container itself, instead its output is passed in as a `--build-arg`. That means that KBFSDocker builds won't work after this is merged in, as long as the KBFSDocker changes aren't merged in. So we should probably wait to merge this until the KBFSDocker stuff is also ready.